docs/custom.md

```
@@ -13,18 +13,18 @@ local Squash = require(...)
 local arr = Squash.array
-local i3 = Squash.int(3)
-local u2 = Squash.uint(2)
+local i24 = Squash.i24()
+local u16 = Squash.u16()
 
 local Serializers = {}
 ```
 
-The below example implementation wastes a few bytes at the end of each array to redundantly record a length that can only be stored once. The type could be changed to `{ { x: number, z: number, id: number } }` but that comes with a performance overhead for so many table creations when deserializing.
+The below example implementation wastes a few bytes at the end of each array to redundantly record a length that can be stored only once. The type could be changed to `{ { x: number, z: number, id: number } }` but that comes with a performance overhead because of so many table creations when deserializing.
 ```lua
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)